issue_06/script.js

```diff
@@ -11,21 +11,27 @@ const contents = [{
 }];
 
 const ul = document.getElementById('js-ul');
-const checkContents = new Promise((resolve) => {
-    resolve(contents);
-});
-checkContents.then((value) => {
+const wait = (sec) => {
+    return new Promise((resolve) => {
+        setTimeout(resolve, sec * 1000);
+    });
+}
+
+async function createLists() {
+    await wait(3);
+
     const fragment = document.createDocumentFragment();
-    for (let i = 0; i < value.length; i++) {
+    for (const item of contents) {
```

Comment:
MUST
It refers to the variable contents instead of the resolved content.

Reply:
A.This is because if you write the array name, you will not receive the resolved value.

Reply:
I understood it to mean that I would receive a resolved value!
Thank you very much :sparkles: